pytripgui/view/README_autogen.txt

```diff
@@ -0,0 +1,2 @@
+pyuic5 main_window.ui > main_window.py
```

Comment:
Can we move it somewhere else, i.e. to issue discussion ?

Reply:
should move into a script... :)